src/website/src/pages/ec/components/forms/select/docs/usage.mdx

```diff
@@ -84,3 +86,67 @@ import { Paragraph, Anatomy, Link } from '@ecl/website-components';
     label="Text field"
     standalone
   />
+
+# Multi-dropdown select
```

Comment:
the wiki is `Multi-select` here ... and I think you've put it correctly having in mind the terms used in the wiki